client-runtime/src/main/java/com/microsoft/rest/v2/BodyResponse.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.rest.v2;
+
+import java.util.Map;
+
+/**
+ * A response to a REST call with a strongly-typed body specified.
+ *
+ * @param <TBody> The deserialized type of the response body.
+ */
+public final class BodyResponse<TBody> extends RestResponse<Void, TBody> {
```

Comment:
Did you create GitHub issues for incorporating this into the generator?

Reply:
https://github.com/Azure/autorest.java/issues/153